src/models/Discount.php

```diff
@@ -341,6 +342,16 @@ public function getPercentDiscountAsPercent(): string
 
         return Craft::$app->formatter->asPercent(0);
     }
+    
+    public function getUserConditions(): array
+    {
+        return [
+          DiscountRecord::CONDITION_USERS_ANY_OR_NONE => Craft::t('commerce', 'Any or none'),
+          DiscountRecord::CONDITION_USERS_INCLUDE_ALL => Craft::t('commerce', 'Include all'),
+          DiscountRecord::CONDITION_USERS_INCLUDE_ANY => Craft::t('commerce', 'Include any'),
```

Comment:
Consider changing to `Include any…` to indicate reliance on additional information.